.github/workflows/release-images.yaml

```
@@ -43,6 +49,18 @@ jobs:
             IMAGE_NAMESPACE=${{ github.repository }} \
             PLATFORM=${{ matrix.platform }}/${{ matrix.arch }} \
             BUILDX_ACTION=--push
+      
+      - name: Sign container image
+        run: |
+          for image in \
+            ghcr.io/${{ github.repository }}/retina-agent:$TAG \
+            ghcr.io/${{ github.repository }}/retina-agent:$TAG-${{ matrix.platform }}-${{ matrix.arch }} \
+            ghcr.io/${{ github.repository }}/retina-init:$TAG \ 
+            ghcr.io/${{ github.repository }}/retina-init:$TAG-${{ matrix.platform }}-${{ matrix.arch }} \
```

```suggestion
            ghcr.io/${{ github.repository }}/retina-agent:$TAG-${{ matrix.platform }}-${{ matrix.arch }} \
            ghcr.io/${{ github.repository }}/retina-init:$TAG-${{ matrix.platform }}-${{ matrix.arch }} \
```
The multi-platform images are created during the `manifests` job, they are not yet available here where we are just building all of the platform-specific images separately.